src/player.js

```diff
@@ -1608,6 +1603,11 @@ export default class Player extends FakeEventTarget {
     const el = this._posterManager.getElement();
     Utils.Dom.addClassName(el, POSTER_CLASS_NAME);
     Utils.Dom.appendChild(this._el, el);
+    // Append playkit-subtitles
+    this._textDisplayEl = Utils.Dom.createElement('div');
+    Utils.Dom.setAttribute(this._textDisplayEl, 'aria-live', 'polite');
+    Utils.Dom.addClassName(this._textDisplayEl, SUBTITLES_CLASS_NAME);
+    Utils.Dom.appendChild(this._el, this._textDisplayEl);
```

Comment:
@Yuvalke what if there is captions at 0 time and preload true. the captions will displayed over the pre-playback screen no?

Reply:
Nothing change for pre-playback it's internal change on playkit on for the order it appends on.
Anyway pre-playback has z-index:1